npm-shrinkwrap.json

```diff
@@ -1,322 +0,0 @@
-{
```

Comment:
This module requires no dependencies for production use, only development usage. Currently, the shrinkwrap is forcing a regular `npm install` to install all of this module's devDependencies. In the following example I made a new directory and ran `npm init` within it, then installed and saved madison in the `package.json`.

<img width="1417" alt="screen shot 2015-12-25 at 4 05 21 pm" src="https://cloud.githubusercontent.com/assets/730443/12004356/d21a4d76-ab21-11e5-893c-9e64a2e3f704.png">